Backpack-SwiftUI/Tests/Slider/SliderTests.swift

```diff
@@ -42,21 +42,6 @@ class SliderTests: XCTestCase {
             .padding()
         )
     }
-    
-    func testRangeSliderWithThumbnailLabels() {
```

Comment:
not possible to test this anymore, as user interaction is needed to make the labels visible :/